### ISSUE TITLE
Is this action used to extract image metadata from the DockerHub registry, or from Git?

### ISSUE THREAD

**Issue by theo-brown** [docker/metadata-action#116]
I'm confused by this action's documentation. It says that it's possible to extract 'tags' and 'labels', but is unclear whether the tags are Git tags (from the Git repository, or tags of remote Docker image stored in the registry.

Does this action allow you to extract metadata from the DockerHub registry?

**Comment by kusur** [comment]
Seems unlikely. Based on my observations, tags are taken from the tags of the git repo. Eg - if v1.0 is the tag in git repo, docker image will be pushed to Docker Hub using v1.0. If  you don't pass any tag, the tag used in Docker Hub will the name of the branch you use in the Github Actions. 
Hope this helps. Please let me know if you have any more questions.

**Comment by crazy-max** [comment]
@theo-brown Yes we are using Git ref and GitHub events to generate tags and labels.

**Comment by theo-brown** [comment]
Please could this be made explicit in the README?
